api/app/Http/Resources/UserResource.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+use Illuminate\Database\Eloquent\Collection;
+
+use App\Models\Department;
+use GraphQL\Experimental\Executor\Collector;
+
+class UserResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array|\Illuminate\Contracts\Support\Arrayable|\JsonSerializable
+     */
+    public function toArray($request)
+    {
+
+        $awardExperiences = AwardExperienceResource::collection($this->awardExperiences);
+        $communityExperiences = CommunityExperienceResource::collection($this->communityExperiences);
+        $educationExperiences = EducationExperienceResource::collection($this->educationExperiences);
+        $personalExperiences = PersonalExperienceResource::collection($this->personalExperiences);
+        $workExperiences = WorkExperienceResource::collection($this->workExperiences);
+
+        $collection = collect();
+        $collection = $collection->merge($awardExperiences);
+        $collection = $collection->merge($communityExperiences);
+        $collection = $collection->merge($educationExperiences);
+        $collection = $collection->merge($personalExperiences);
+        $collection = $collection->merge($workExperiences);
+
+        return [
+            'id' => $this->id,
+            'sub' => $this->sub,
+            'roles' => $this->roles,
+            'firstName' => $this->first_name,
+            'lastName' => $this->last_name,
+            'email' => $this->email,
+            'telephone' => $this->telephone,
+            'preferredLang' => $this->preferred_lang,
+            'currentProvince' => $this->current_province,
+            'currentCity' => $this->current_city,
+            'citizenship' => $this->citizenship,
+            'armedForcesStatus' => $this->armed_forces_status,
+            'languageAbility' => $this->language_ability,
+            'lookingForEnglish' => $this->looking_for_english,
+            'lookingForFrench' => $this->looking_for_french,
+            'lookingForBilingual' => $this->looking_for_bilingual,
+            'bilingualEvaluation' => $this->bilingual_evaluation,
+            'comprehensionLevel' => $this->comprehension_level,
+            'writtenLevel' => $this->written_level,
+            'verbalLevel' => $this->verbal_level,
+            'estimatedLanguageAbility' => $this->estimated_language_ability,
+            'isGovEmployee' => $this->is_gov_employee,
+            'hasPriorityEntitlement' => $this->has_priority_entitlement,
+            'govEmployeeType' => $this->gov_employee_type,
+            'department' => $this->department ? (new DepartmentResource(Department::find($this->department))) : null,
+            'currentClassification' => (new ClassificationResource($this->currentClassification)),
```

Comment:
I think it has something to do with the relationship attribute matching the database column.

- **Department**
  - Column: `department`
  - Relationship: `department`
- **Classification**
  - Column: `current_classification`
  - Relationship: `currentClassification`

When creating relationships I usually create it like this to avoid conflicts:

- Column: `department_id`
- Relationship: `department`

Reply:
Ahh gotcha 👍